--- lib/BUILD
+++ lib/BUILD
@@ -5,6 +5,7 @@
 
 load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 
 # Load sources/headers/tests lists.
 load(
@@ -61,7 +62,7 @@
     "libjxl_test_timeouts",
 )
 
-DEFAULT_VISIBILITY = ["//:__subpackages__"]
+DEFAULT_VISIBILITY = ["//visibility:public"]
 
 DEFAULT_COMPATIBILITY = []
 
